agent/stats/container.go

```diff
@@ -107,6 +108,34 @@ func (container *StatsContainer) collect() {
 	}
 }
 
+func (container *StatsContainer) getApiContainer(dockerID string) (*apicontainer.Container, error) {
+	if container.resolver == nil {
+		return nil, nil
```

Comment:
Is container's resolver being nil considered an error case? If so, should an error be returned here?

Reply:
good call, I will change this